README.md

```diff
@@ -132,32 +128,110 @@ and some optional params can be provided as well:
     modbus_read[{$MODBUS_PORT},32,4,3,uint32,BE,0]
     modbus_read[enc://192.168.1.1,1,6,1]
     modbus_read[tcp://192.168.1.1:5000,1,5,1]
+    modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,1,100*bit]
+    modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,1,10*double]
+    modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,3,2*uint16+skip+float]
+    modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,3,2*uint16+double+float*3,MLE]
 ```
-  
-**Additional item parameters:**  
-Item type:  
-  
-* *Simple check* if you load module on zabbix_server or zabbix_proxy  
-* *Zabbix Agent* or *Zabbix Agent(active)* if you load module on zabbix_agent  
 
-  
+
+## 5. Bulk Data Collection
+It is now possible to get as many Modbus registers as needed with just a single command and return it as the JSON object. JSON object is then processed by Zabbix (version 3.4 or newer is required) and all single values routed to different dependent items.   
+
+In order to get data in bulk, you first need to define which registers you want to retrieve using datatype expression. 
+
+### 5.1. The Simplest Example
+The simplest datatype expression would be 
+`2*uint16` that you write in the key: `modbus_read[{$MODBUS_PORT},{$MODBUS_SLAVE},10,3,2*uint16]`  
+This expression is quite simple - it just retrieves two unsigned integers starting from register 10. Here is what you would get:
+```
+{
+  "10":123,
+  "11":321
+}
```

Comment:
Actually, the result won't be pretty-printed like that. Should we mention it?

Reply:
That's ok,  Here it's easier to read like that.